### DISCUSSION TITLE
Archiving Gitlab repositories

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-core#10864] [Planning]
I want to push the git repositories that we have in GitLab to GitHub.

These repositories: 

https://gitlab.com/micronaut-projects/eureka-server
https://gitlab.com/micronaut-projects/micronaut-graal-tests
https://gitlab.com/micronaut-projects/micronaut-graal-tests-scheduler
https://gitlab.com/micronaut-projects/upgrade-micronaut-version

to GitHub

https://github.com/micronaut-projects/gitlab-eureka-server
https://github.com/micronaut-projects/gitlab-micronaut-graal-tests
https://github.com/micronaut-projects/gitlab-micronaut-graal-tests-scheduler
https://github.com/micronaut-projects/gitlab-upgrade-micronaut-version

Then, archive these GitHub repositories to preserve them and then delete the GitLab organization. 

We are no longer using GitLab. We used it to test our Graal integration. 

@micronaut-projects/core-developers thoughts?

**Comment by graemerocher** [comment]
fine by me

**Comment by sdelamo** [comment]
I have imported the repositories and archive them: 

https://github.com/micronaut-projects/gitlab-eureka-server
https://github.com/micronaut-projects/gitlab-micronaut-graal-tests
https://github.com/micronaut-projects/gitlab-micronaut-graal-tests-scheduler
https://github.com/micronaut-projects/gitlab-upgrade-micronaut-version
